docs/volume-management.md

```diff
@@ -172,3 +172,18 @@ Suspended volumes can be resumed once they are fixed. Upon resuming, the corresp
 ```sh
 > kubectl directpv resume volumes --nodes node-1 --drives dm-3
 ```
+
+## SELinux in OpenShift:
+
+If you encounter the `relabel failed` error after executing the `suspend` or `resume` commands, you should set `spc_t` at the Tenant level Specification, as demonstrated below:
+
+```yaml
+kind: Tenant
+spec:
+  pools:
+  - containerSecurityContext:
+      seLinuxOptions:
+        type: spc_t
```

Comment:
Is there a consequence to setting this at initial setup? E.g. should the operator simply do this in general?

Reply:
I think Operator should only do this for OpenShift if needed, but that extra logic could goes into a different PR.